conda_smithy/ci_register.py

```diff
@@ -178,48 +178,38 @@ def add_project_to_travis(user, project):
     headers = travis_headers()
     endpoint = 'https://api.travis-ci.org'
 
-    url = '{}/hooks'.format(endpoint)
+    url = '{}/{}/{}/key'.format(endpoint, user, project)
```

Comment:
This is an interesting idea. I like it. Though we could just drop the `key` part.